### ISSUE TITLE
Use content type category_list in blocks

### ISSUE THREAD

**Issue by fly85** [sulu/sulu#3537]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.6.4
| Browser Version | *

#### Actual Behavior

Adding property with the content type category_list within block is not working on homepage template.
The same code on other template is working.
Adding the content type category_list within a block generate the following JS error in the administration:
`categoryList.js?v=develop:45 Uncaught TypeError: Cannot read property 'map' of null`

#### Expected Behavior

Allow to use category_list content type in block on homepage

#### Steps to Reproduce

Homepage template:
```
<?xml version="1.0" ?>
<template xmlns="http://schemas.sulu.io/template/template"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">

    <key>homepage</key>

    <view>templates/homepage</view>
    <controller>SuluWebsiteBundle:Default:index</controller>
    <cacheLifetime>2400</cacheLifetime>

    <meta>
        <title lang="en">Homepage</title>
        <title lang="de">Startseite</title>
    </meta>

    <properties>
        <section name="highlight">
            <properties>
                <property name="title" type="text_line" mandatory="true">
                    <meta>
                        <title lang="en">Title</title>
                        <title lang="de">Titel</title>
                    </meta>
                    <params>
                        <param name="headline" value="true"/>
                    </params>

                    <tag name="sulu.rlp.part"/>
                </property>

                <property name="url" type="resource_locator" mandatory="true">
                    <meta>
                        <title lang="en">Resourcelocator</title>
                        <title lang="de">Adresse</title>
                    </meta>

                    <tag name="sulu.rlp"/>
                </property>
            </properties>
        </section>
        
        <block name="testimonials" default-type="testimonial" minOccurs="1">
            <types>
                <type name="testimonial">
                    <properties>
                        <property name="category" type="category_list">
                            <meta>
                                <title lang="en">Category</title>
                            </meta>
                        </property>
                        <property name="quote" type="text_area" mandatory="true">
                            <meta>
                                <title lang="en">Quote</title>
                            </meta>
                        </property>
                        <property name="quote_author" type="text_line" mandatory="true">
                            <meta>
                                <title lang="en">Author</title>
                            </meta>
                        </property>
                    </properties>
                </type>
            </types>
        </block>
    </properties>
</template>
```
Go the homepage administration

**Comment by wachterjohannes** [comment]
@fly85 I can reproduce it and found a fix. The PR is also done - so the next HOTFIX release should fix it for you.

**Comment by danrot** [comment]
Fixed in #3541
